rboot-sampleproject/rom0.ld

```diff
@@ -207,4 +207,4 @@ SECTIONS
 }
 
 /* get ROM code address */
-INCLUDE "../ld/eagle.rom.addr.v6.ld"
+INCLUDE "../rboot/eagle.rom.addr.v6.ld"

```

Comment:
ok, you must be using a different version to me then, my preferred option would be to upload a copy of the original file to the sample project, so it doesn't look like some odd rboot dependancy